index.html.in

```diff
@@ -217,22 +217,22 @@
                                   </div>
                               </div>
 
-                              <div class="modal-backdrop" id="fileDropBackdrop" style="display: none; z-index: 1000;">
-                                <div class="well well-lg" style="text-align: center; position: absolute; top: 35%; left: 35%;">
-                                    <h1 style="font-weight: bold;" data-text="Drop_Document">Drop your document</h1>
+                              <div class="modal-backdrop" id="fileDropBackdrop">
+                                <div class="well well-lg">
+                                    <h1 data-text="Drop_Document">Drop your document</h1>
                                 </div>
-                                <div class="modal-backdrop" id="fileDropMask" style="opacity: 0; z-index: 10000;"></div>
+                                <div class="modal-backdrop" id="fileDropMask"></div>
```

Comment:
Where is `#fileDropMask` coming from?

Reply:
I didn't understand.

Reply:
My question is why you are calling it that.